src/components/document-metadata-modal.yml

```diff
@@ -98,9 +98,9 @@ documentMetadataModal:
   noAvailableRooms:
     en: No available rooms
     de: Keine Räume verfügbar
-  maintenanceSettingsHeader:
-    en: Maintenance settings
-    de: Maintenance-Einstellungen
+  maintainerSettingsHeader:
+    en: Maintainer settings
+    de: Maintainer-Einstellungen
```

Comment:
Renamed this to reflect the role instead of the page name, also with this change, the word "maintenance" only refers to the maintenance we do at the start of the application ("maintenance mode", where we ensure database collections and basic CDN directories). Nowhere else in the whole project does the word "maintenance" refer to the role "maintainer" anymore with this change.